pkg/controller/kabaneroplatform/landing.go

```diff
@@ -24,10 +28,10 @@ import (
 var kllog = rlog.Log.WithName("kabanero-landing")
 
 // Deploys resources and customizes to the Openshift web console.
-func deployLandingPage(k *kabanerov1alpha1.Kabanero, c client.Client) error {
+func deployLandingPage(k *kabanerov1alpha1.Kabanero, c client.Client, reqLogger logr.Logger) error {
```

Comment:
This is one of those parts that has it's own logr to avoid making the trace/failure look like it is coming from the caller as opposed to this part. I find it easier to debug when the traces show, prominently, the part where the failure happened. Having said that, if you don't want to use the existing logr, you might want to delete it in favor of what you are using.

Reply:
I did not realize that this part had its own logger, I should have looked more closely.  I will say though that the logger that I am passing in is modified to show the current request:
```go
reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
```
I'm not opposed to using a different logger, but I think the team should probably talk about how logging is going to work, as I see the benefit of both modifying the logger to display the current request, but also having the logger indicate which part it's in.